pom.xml

```diff
@@ -20,7 +20,7 @@
     <properties>
         <mavenVersion>3.8.1</mavenVersion>
         <mavenPluginPluginVersion>3.2</mavenPluginPluginVersion>
-        <provisioVersion>0.1.40</provisioVersion>
+        <provisioVersion>1.0.18</provisioVersion>
         <takari.javaSourceVersion>1.8</takari.javaSourceVersion>
```

Comment:
Any clue what this references and would it need an upgrade ?

Reply:
It's getting referenced just a few lines below for setting language level. Probably once we want to change the language level we will have to update this property as well.